.github/workflows/rust.yml

```diff
@@ -21,8 +21,6 @@ jobs:
     - name: Check format
       run: cargo fmt -- --check
     - name: Cargo clippy check
-      env:
-        RUSTFLAGS: -D warnings
```

Comment:
Doesn't removing this make the `clippy` check ineffective? The CI will pass even if there are clippy warnings.